src/webview_provider/queryResultPanel.ts

```
@@ -433,9 +433,9 @@ export class QueryResultPanel extends AltimateWebviewProvider {
             const perspectiveTheme = workspace
               .getConfiguration("dbt")
               .get("perspectiveTheme", "Vintage");
-            const queryBookmarksEnabled = workspace
+            const queryHistoryDisabled = workspace
               .getConfiguration("dbt")
-              .get("enableQueryBookmarks", false);
+              .get("disableQueryHistory", false);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)